fastavro/_write.pyx

```diff
@@ -48,7 +50,7 @@ cdef inline write_boolean(bytearray fo, bint datum):
     fo += ch_temp[:1]
 
 
-cdef inline write_int(bytearray fo, datum):
+cdef inline write_int(bytearray fo, long long datum):
```

Comment:
I just realized it's probably preferred to use the typename convention:

```suggestion
cdef inline write_int(bytearray fo, long64 datum):
```

Reply:
Agreed